Puzzlepart/did
# NN + NB language improvements

### Your checklist for this pull request
 
### Review checklist

### Review theme song
🎵  [Side Brok - Heimegut (Sagtanns Crocket remix)](https://open.spotify.com/track/2TowhqtJFSoHh6pGapW4iw) 🎵 

### Description
Language improvements.  
Mostly Nynorsk, but some Bokmål as well.  
Nynorsk improvements approved by @paalolav 

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Check each of the changed strings in `no.json` and `nb.json` and agree to the changes

### Related discussions
[#did on Slack](https://puzzlepart.slack.com/archives/GQK65AT0S/p1629453366002900)
